test/kubetest/k8s.go

```diff
@@ -718,6 +718,21 @@ func (k8s *K8s) DescribePod(pod *v1.Pod) {
 	}
 }
 
+//ShouldNotRestart checks that pod should not restart
+func (k8s *K8s) ShouldNotRestart(pod *v1.Pod, secondsForWait time.Duration) error {
```

Comment:
We can assert inside this function. Please take a look at field `g` of structure `k8s`.